### PR TITLE
TELCODOCS-111: Content for Fujitsu iRMC baseboard management controller

### PR DESCRIPTION
**Pull request by johnwilkins**
Contains iRMC docs for Fujitsu BMC. Docs reviewed by Fujitsu.

Fixes: [TELCODOCS-111](https://issues.redhat.com/browse/TELCODOCS-111)

See https://issues.redhat.com/browse/TELCODOCS-111 for additional details.

Preview URL: https://deploy-preview-33751--osdocs.netlify.app/?utm_source=github&utm_campaign=bot_dp

For release(s): 4.8
Signed-off-by: John Wilkins <jowilkin@redhat.com>

### CODE REVIEW

File: modules/ipi-install-bmc-addressing-for-fujitsu.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+.BMC address formats for Fujitsu hardware
+[frame="topbot",options="header"]
+|====
+|Protocol|Address Format
+|iRMC| `irmc://<out-of-band-ip>`
+|IPMI| `ipmi://<out-of-band-ip>`
+|====
+
+See the following section for additional details.
```
Comment (ahardin-rh) [inline modules/ipi-install-bmc-addressing-for-fujitsu.adoc:33]: I would remove this line because the details directly follow anyway

**Comment by ahardin-rh** [comment]
Holding while we wait for confirmation from QE and Telco DPM/CS, as this deals with 3rd party content that has not been QE reviewed by Red Hat.

**Comment by hase1128** [comment]
Hi, I'm Fujitsu engineer.
Thank you for the draft. I confirmed this document looks good, thanks!

File: installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc [hunk 2]
```diff
@@ -28,6 +28,10 @@ include::modules/ipi-install-bmc-addressing-for-dell.adoc[leveloffset=+1]
 
 include::modules/ipi-install-bmc-addressing-for-hpe.adoc[leveloffset=+1]
 
+ifeval::[{product-version} > 4.7]
+include::modules/ipi-install-bmc-addressing-for-fujitsu.adoc[leveloffset=+2]
```
Comment (sferich888) [inline installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc:32]: Let's re-title the module to not list a customer names, but to make it more clear that this is for a specific HW that a customer can purchase / use.

File: modules/ipi-install-bmc-addressing-for-fujitsu.adoc [hunk 3]
```diff
@@ -0,0 +1,53 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id='bmc-addressing-for-fujitsu_{context}']
+
+= BMC addressing for Fujitsu
```
Comment (sferich888) [inline modules/ipi-install-bmc-addressing-for-fujitsu.adoc:6]: Lets make it clear that this is for Fujitsu HW not Fujitsu the customer/partner of ours.

File: modules/ipi-install-bmc-addressing-for-fujitsu.adoc [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: <host name>
+        role: <master | worker>
+        bmc:
+          address: <address>
```
Comment (sferich888) [inline modules/ipi-install-bmc-addressing-for-fujitsu.adoc:18]: I feel like this needs a footnote or something to connect it to the 'protocol' scheme statements being made above and below.

**Review by sferich888 — APPROVED** [review]
At the end of the day what this PR is trying to highlight is that the 'address' field for the 'bmc' can use one of 2 protocols and that the protocol needs to be specified in the 'address' setting. 

IE: `irmc://<out-of-band-ip>` OR `ipmi://<out-of-band-ip>` 

/lgmt

File: modules/ipi-install-bmc-addressing-for-fujitsu.adoc [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+.BMC address formats for Fujitsu hardware
+[frame="topbot",options="header"]
+|====
+|Protocol|Address Format
+|iRMC| `irmc://<out-of-band-ip>`
+|IPMI| `ipmi://<out-of-band-ip>`
+|====
+
+.iRMC for Fujitsu
```
Comment (mbearer) [inline modules/ipi-install-bmc-addressing-for-fujitsu.adoc:33]: @sferich888  Does your comment above apply here too? Should this be .iRMC for Fujitsu hardware?

File: installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc (hunk 2 above)
Comment (johnwilkins) [reply installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc]: Renamed modules to include protocols after vendor names. Dell iDRAC, HPE iLO, Fujitsu iRMC.

**Comment by vikram-redhat** [comment]
All feedback has now been resolved. @ahardin-rh if there is nothing else, it is ok to merge.

PM, CCS, QE and PX acks have been received via email.

**Comment by ahardin-rh** [comment]
Thanks, all!

**Pull request merged**